packages/core/upload/admin/src/components/AssetCard/UploadingAssetCard.tsx

```diff
@@ -15,11 +15,11 @@ import {
 import { useIntl } from 'react-intl';
 import { styled } from 'styled-components';
 
-// TODO: replace this import with the import from constants file when it will be migrated to TS
-import { AssetType } from '../../newConstants';
+import { AssetType } from '../../constants';
 import { useUpload } from '../../hooks/useUpload';
-import { getTrad } from '../../utils';
-import { UploadProgress } from '../UploadProgress';
+import { getTrad } from '../../utils/utils';
+import { UploadProgress } from '../UploadProgress/UploadProgress';
```

Comment:
just in case I will ask, is it common that we nest imports such as UploadProgress/UploadProgress?
would it be handier to create an index file for that one ? or we do so its easier to find files?

Reply:
I had to replace every single index file with a named file to follow the eslint rules defined in the configuration file...I did it to follow the same rules we were using in the Review-workflow plugin

Reply:
I really do not agree with imports like this. It is unnecessarily redundant. I personally don't see the issue with index files 🤷 maybe someone could fill me in on why we have such a strict rule against them?

If we don't want to use them then the folder shouldn't be the same name as the component to avoid these types of imports. Here this is the only component in the folder. The folder is apparently there for a tests folder. But does that make sense? Should we just flatten this to an UploadProgress.tsx _file_ and put the tests in a tests folder at the root of components? I think that could be done for a lot of components.

I think we should probably discuss how we want to organize components in an FE sync or something so we are all on the same page.